Source/Eto/Forms/Menu/MenuItemCollection.cs

```diff
@@ -134,15 +134,9 @@ public void AddSeparator(int order = 0)
 		/// <param name="items">Items to add.</param>
 		public void AddRange(IEnumerable<MenuItem> items)
 		{
-			var list = Items as List<MenuItem>;
```

Comment:
These changes should already be fixed in the develop branch, not sure why they're showing on the pull.  Perhaps you can rebase then force push to the branch?